tidb-cloud/set-up-sink-private-endpoint.md

```diff
@@ -42,6 +42,24 @@ If the Private Endpoint Service is not available for your downstream service, fo
 
 </div>
 
+<CustomContent plan="premium">
+
+<div label="Alibaba Cloud">
+
+If your changefeed downstream service is hosted on Alibaba Cloud, collect the following information:
+
+- The name of the Private Endpoint Service for your downstream service
+- The availability zones (AZs) where your downstream service is deployed
+
+It is necessary to whitelist TiDB Cloud's Alibaba Cloud account ID for your endpoint service to grant the TiDB Cloud VPC access.
```

Comment:
```suggestion
To grant TiDB Cloud VPC access, you must add the TiDB Cloud's Alibaba Cloud account ID to your endpoint service allowlist.
```

Reply:
https://developers.google.com/style/word-list#blacklist
Don't use blacklist, whitelist, and graylist.